pkg/Microsoft.DotNet.ILCompiler/TargetSpecific/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -0,0 +1,14 @@
+<!-- The ILCompiler package is produced by including built binaries and executables from the OS-Specific builds.
+  These contents are defined in Microsoft.Dotnet.ILCompiler.Common.props - the purpose of this file is to force the build process to produce runtime-specific packages. -->
```

Comment:
`Microsoft.Dotnet.ILCompiler.Common.props` does not need to be separate file now. It can be just folded into this one.